### DISCUSSION TITLE
Laravel 10 Support

### DISCUSSION THREAD

**Discussion by sharathdn1** [RTippin/messenger#57] [General]
Hello @RTippin,

Does this package works with Laravel 10 ?

**Comment by RTippin** [comment]
@sharathdn1 I am planning to support laravel 10 with the 1.x of this package. I have been unable to do so as I am waiting for another dependency to be updated. See: https://github.com/Intervention/imagecache/pull/134

I have already upgraded to L10 and tested the code on the master branch, but I had to uninstall the image cache dependency to get that far. See: https://github.com/RTippin/messenger/tree/master and https://github.com/RTippin/messenger/actions/runs/4176433466

I'll probably hold out for another week, and if the above issue is not resolved by then, I will find another path and publish the L10 supported version. Only once I've tagged a new release for this package can I also update my sub packages (messenger-faker, messenger-bots, messenger-ui).

**Reply by sharathdn1** [reply]
Hi @RTippin, Thanks for the update.
Hope to see the the support from the another package too.. 

Thanks for your contribution.

**Reply by RTippin** [reply]
@sharathdn1 The above has been resolved.
